File: src/vehicle/rental/system/Invoice/Invoice.java
package vehicle.rental.system.Invoice;

import java.math.BigDecimal;
import java.time.LocalDate;

public record Invoice(LocalDate date,
                      String customerName,
                      String vehicleBrand,
                      String vehicleModel,
                      LocalDate startDate,
                      LocalDate endDate,
                      long reservedDays,
                      LocalDate actualReturnDate,
                      long actualDays,
                      BigDecimal dailyInsuranceCost,
                      BigDecimal dailyRentalCost,
                      BigDecimal totalRentalCost,
                      BigDecimal totalInsuranceCost
) {
}

File: src/Invoice/Invoice.cs
namespace VehicleRentalSystem.Invoice;

public record Invoice(
    DateOnly Date,
    string CustomerName,
    string VehicleBrand,
    string VehicleModel,
    DateOnly StartDate,
    DateOnly EndDate,
    long ReservedDays,
    DateOnly ActualReturnDate,
    long ActualDays,
    decimal DailyInsuranceCost,
    decimal DailyRentalCost,
    decimal TotalRentalCost,
    decimal TotalInsuranceCost
);
